src/View/SerializedView.php

```diff
@@ -84,8 +85,23 @@ public function render($view = null, $layout = null)
         }
 
         if ($serialize !== false) {
-            return $this->_serialize($serialize);
-        } elseif ($view !== false && $this->_getViewFileName($view)) {
+            $result = $this->_serialize($serialize);
+            if ($result === false) {
+                if (Configure::read('debug') && in_array('trace', $serialize)) {
```

Comment:
Instead of trying to hack out the trace, could we just trigger the error and let people know their data is not going to work?